ledger/participant-integration-api/src/main/resources/db/migration/h2database-appendonly/V1__Append_only_schema.sql

```diff
@@ -55,13 +55,13 @@ CREATE TABLE participant_user_rights (
                                                                     THEN for_party
                                                                     ELSE ''
                                                              END),
-    granted_at          BIGINT          NOT NULL DEFAULT CAST((1000 * 1000 * EXTRACT(epoch FROM CURRENT_TIMESTAMP)) AS BIGINT),
+    granted_at          BIGINT          NOT NULL,
     UNIQUE (user_internal_id, user_right, for_party2)
 );
 
-INSERT INTO participant_users(user_id, primary_party) VALUES ('participant_admin', NULL);
-INSERT INTO participant_user_rights(user_internal_id, user_right, for_party)
-    SELECT internal_id, 1, NULL
+INSERT INTO participant_users(user_id, primary_party, created_at) VALUES ('participant_admin', NULL, CAST((1000 * 1000 * EXTRACT(epoch FROM CURRENT_TIMESTAMP)) AS BIGINT));
```

Comment:
@nmarton-da @rautenrieth-da What would be the place in Scala code to move this insertion?

Reply:
this timestamp does not convey much information, I would just go with 0.
My point: being able to drop all (and especially Oracle) timestamp generation / conversion etc

Reply:
Ok